lib/matplotlib/widgets.py

```diff
@@ -2102,12 +2106,13 @@ def __init__(self, ax, onselect, direction, minspan=0, useblit=False,
         self.new_axes(ax)
 
         # Setup handles
-        props = dict(color=rectprops.get('facecolor', 'r'))
-        props.update(cbook.normalize_kwargs(handle_props, Line2D._alias_map))
+        handle_props = {
+            'color': (props or {}).get('facecolor', 'r'),
```

Comment:
`props` can't be `None` due to line 2080?

Reply:
Yes, indeed!